ribasim_qgis/widgets/dataset_widget.py

```diff
@@ -208,6 +212,32 @@ def add_selection_to_qgis(self) -> None:
         for item in selection:
             self.add_item_to_qgis(item)
 
+    @staticmethod
+    def add_relationship(from_layer, to_layer_id, name, fk="node_id") -> None:
+        rel = QgsRelation()
+        rel.setReferencingLayer(from_layer.id())
+        rel.setReferencedLayer(to_layer_id)
+        rel.setName(name)
+        rel.setStrength(rel.RelationStrength.Composition)  # type: ignore
```

Comment:
Why is the type ignored in this case? Version mismatch between the stubs?

Reply:
The stubs do not seem to be aware of enums linked in a class.